tests/integration/run-locally.sh

```diff
@@ -3,19 +3,21 @@
 set -e
 
 expandpath() {
-  CDPATH= cd -- "$1" && pwd -P
+  CDPATH=cd -- "$1" && pwd -P
```

Comment:
The space there was intentional. We do not want to assign to `CDPATH`. Another way would be `CDPATH="" cd -- ...`